packages/server/src/model/serlo.ts

```diff
@@ -56,8 +56,13 @@ export function createSerloModel({
       decoder: DatabaseLayer.getDecoderFor('UuidQuery'),
       enableSwr: true,
       getCurrentValue: async (payload: DatabaseLayer.Payload<'UuidQuery'>) => {
-        const uuid = await DatabaseLayer.makeRequest('UuidQuery', payload)
-        return isSupportedUuid(uuid) ? uuid : null
+        const uuid = await DatabaseLayer.makeRequest('UuidQuery', payload) as AbstractUuid
```

Comment:
```suggestion
        const uuid = await DatabaseLayer.makeRequest('UuidQuery', payload) as AbstractUuid
```
I want to have a better solution here than `as AbstractUuid`. But makeRequest is an unknown Promise.